### PR TITLE
Change prebid to depend on the initialisation of the prebid library

### PR DESCRIPTION
**Pull request by frankie297**
## What does this change?
Changes prebid to depend on the initialisation of the prebid library

### Tested

- [ ] Locally
- [x] On CODE (optional)

### CODE REVIEW

**Review by kelvin-chappell — APPROVED** [review]
Much better! 👍

File: static/src/javascripts/projects/commercial/modules/dfp/prepare-prebid.js [hunk 1]
```diff
@@ -14,22 +14,25 @@ const isGoogleWebPreview: () => boolean = () =>
             navigator.userAgent.indexOf('googleweblight') > -1)
     );
 
+let moduleLoadResult = Promise.resolve();
+
 if (!isGoogleWebPreview()) {
-    import(/* webpackMode: "eager" */ 'prebid.js/build/dist/prebid');
+    moduleLoadResult = import('prebid.js/build/dist/prebid');
```
Comment (janua) [inline static/src/javascripts/projects/commercial/modules/dfp/prepare-prebid.js]: @SiAdcock This will now end up in it's own WebPack fragment.

Comment (janua) [reply static/src/javascripts/projects/commercial/modules/dfp/prepare-prebid.js]: @frankie297 Could we add the magic comment for naming the fragment from the documentation?

The fragment is coming out as `graun.38.js` at the moment. We should probably call it Prebid.js.

See [`webpackChunkName`](https://webpack.js.org/api/module-methods/#magic-comments).

Comment (frankie297) [reply static/src/javascripts/projects/commercial/modules/dfp/prepare-prebid.js]: 👍

**Comment by janua** [comment]
:+1: :confetti_ball:

**Pull request merged**